### ISSUE TITLE
ReferenceError: regeneratorRuntime is not defined

### ISSUE THREAD

**Issue by stefanheine** [thenativeweb/runfork#3]
Hi Golo,

thanks for releasing the updated version, but the new version is throwing an error:

```
     ReferenceError: regeneratorRuntime is not defined
      at node_modules/runfork/dist/runfork.js:36:48
      at runfork (node_modules/runfork/dist/runfork.js:91:4)
```

according to some google findings, this seems to be related to babel settings while building your distribution package.
  Stefan

**Comment by goloroden** [comment]
Fixed with newly released `1.0.1`.

**Comment by stefanheine** [comment]
thanks!
